controllers/ticketingController.js

```diff
@@ -32,8 +33,21 @@ const cancelReservedSeat = catchAsync(async (req, res) => {
   return res.status(200).json({ message: 'TICKET CANCEL SUCCESSFULLY' });
 });
 
+const getTicketingDetails = catchAsync(async (req, res) => {
+  const userId = req.user;
+
+  if (!userId) {
+    throw new Error('keyErr');
+  }
```

Comment:
```suggestion
  if (!userId) throw new Error('keyErr');
```
실행문 중괄호(`{}`) 없이 한 줄로 깔끔하게 나타낼 수 있습니다.